docs/copilot/guides/agents-tutorial.md

```diff
@@ -0,0 +1,129 @@
+---
+ContentId: 8f2c9a1b-3d4e-5f6a-7b8c-9d0e1f2a3b4c
+DateApproved: 12/10/2025
+MetaDescription: Get started with different types of agents in VS Code to run tasks locally, in the background, or in the cloud. Hand off work across agents to use what works best for your workflow.
+MetaSocialImage: ../images/shared/github-copilot-social.png
+Keywords:
+- ai
+- agents
+- background agent
+- cloud agent
+- copilot coding agent
+- copilot cli
+- tutorial
+---
+
+# Tutorial: Work with agents in VS Code
+
+This tutorial walks you through using different types of agents in Visual Studio Code. You build a todo app from scratch, add a theme toggle, and redesign the layout by delegating work across local, plan, background, and cloud agents.
+
+> [!TIP]
+> If you don't yet have a Copilot subscription, you can use Copilot for free by signing up for the [Copilot Free plan](https://github.com/github-copilot/signup) and get a monthly limit of completions and chat interactions.
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+* [Visual Studio Code installed on your computer](/download)
+* [A GitHub account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github) (for cloud agent workflow)
+* [A GitHub Copilot subscription](/docs/copilot/setup.md)
+
+## Step 1: Use a local agent to scaffold an app
+
+In this step, you'll use a local agent to create the initial todo app structure. Local agents are perfect for interactive tasks where you want immediate feedback and results, such as scaffolding a new project or planning a new feature.
```

Comment:
Super nit feedback: 
- I love how this reads, though maybe we avoid "perfect"
- Should we avoid mentioning "planning" here explicitly, if we're going to touch on the plan agent instead in the next section?


```suggestion
In this step, you'll use a local agent to create the initial todo app structure. Local agents are ideal for interactive tasks where you want immediate feedback and results, such as scaffolding a new project or iterating on a new feature.
```